examples/neo-place/client/qr.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
```

Comment:
(It's not here)
question: Shall we also delete the QR code png? Do we still need it?